client/src/main/scala/skuber/api/package.scala

```diff
@@ -45,7 +45,26 @@ package object client {
      token: Option[String] = None,
      userName: Option[String] = None,
      password: Option[String] = None
-   )
+   ) {
+     override def toString : String = {
+       var result = new StringBuilder("AuthInfo(")
+       clientCertificate.getOrElse("") match {
+         case Left(cert) => result ++= "clientCertificate=" + cert + " "
+         case Right(cert) => result ++= "clientCertificate=<PEM masked> "
```

Comment:
Can we change this to read:

```
case Left(certPath: String) => result ++= "clientCertificate=" + certPath + " "
case Right(_: Array[Byte]) => result ++= "clientCertificate=<PEM masked> "
```

That makes it more clear what we are printing and what we are masking.